tests/Catalog/OptionBuilderTest.php

```diff
@@ -0,0 +1,100 @@
+<?php
+declare(strict_types=1);
+
+namespace TddWizard\Fixtures\Catalog;
+
+use TddWizard\Fixtures\Catalog\OptionBuilder;
+use TddWizard\Fixtures\Catalog\OptionFixture;
+use TddWizard\Fixtures\Catalog\OptionFixtureRollback;
+use Magento\Catalog\Model\Product;
+use Magento\Eav\Api\AttributeOptionManagementInterface;
+use Magento\Eav\Model\Entity\Attribute\OptionFactory;
+use Magento\Eav\Model\ResourceModel\Entity\Attribute\Option as OptionResource;
+use Magento\TestFramework\Helper\Bootstrap;
+use PHPUnit\Framework\TestCase;
+
+/**
+ * @magentoAppIsolation enabled
+ * @magentoDbIsolation enabled
+ */
+class OptionBuilderTest extends TestCase
+{
+    private $options = [];
+
+    /** @var AttributeOptionManagementoptionManagemente */
+    private $optionManagement;
+
+    /** @var OptionResource */
+    private $optionResourceModel;
+
+    /** @var OptionFactory */
+    private $optionFactory;
+
+    protected function setUp(): void
+    {
+        $objectManager = Bootstrap::getObjectManager();
+        $this->optionManagement = $objectManager->get(AttributeOptionManagementInterface::class);
+        $this->optionFactory = $objectManager->get(OptionFactory::class);
+        $this->optionResourceModel = $objectManager->get(OptionResource::class);
+    }
+
+    protected function tearDown(): void
+    {
+        if (!empty($this->options)) {
+            foreach ($this->options as $optionFixture) {
+                OptionFixtureRollback::create()->execute($optionFixture);
```

Comment:
I'd recommend to use `$optionFixture->rollback()` instead now, which you already implemented. In #52 I made fixture usage easier. An `OptionFixturePool` would be nice as well, to rollback all at once. None of this is a merge blocker though

Reply:
I added the `OptionFixturePool`.